marklogic-data-hub/src/main/resources/ml-modules/root/marklogic-data-hub-spark-connector/initializeJob.api

```diff
@@ -2,7 +2,7 @@
   "functionName" : "initializeJob",
   "desc" : "Creates a job document with the given sparkMetadata as a property",
   "params" : [ {
-    "name" : "sparkMetadata",
+    "name" : "externalMetadata",
```

Comment:
Let's add a "desc" to this to make it clear what the purpose of this is. How about: "JSON object that, if not null, will be added to the job document with a key of 'externalMetadata'".